package/MDAnalysis/analysis/hbonds/hbond_analysis.py

```diff
@@ -479,31 +458,31 @@ def __init__(self, universe, selection1='protein', selection2='all', selection1_
             Selection string for second selection ['all']
         selection1_type : str (optional)
             Selection 1 can be 'donor', 'acceptor' or 'both'. Note that the
-            value for *selection1_type* automatically determines how
-            *selection2* handles donors and acceptors: If *selection1* contains
-            'both' then *selection2* will also contain *both*. If *selection1*
-            is set to 'donor' then *selection2* is 'acceptor' (and vice versa).
+            value for `selection1_type` automatically determines how
```

Comment:
I haven't changed this yet because this is how numpy docs are written (IIRC) and it is consistent with the Napoleon formatting.